common/common.iml

```diff
@@ -7,5 +7,6 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="scala-sdk-2.12.8" level="application" />
```

Comment:
This change may not be needed, but if it is, I'd imagine this would need to be 'provided' or 'compile-only' scope. Otherwise scala sdk would leak into runtime, conflicting with scala plugin.